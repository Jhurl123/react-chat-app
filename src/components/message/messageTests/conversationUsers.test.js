import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import ConversationUsers from '../conversationUsers'
import localStorage from './localStorage'

window.localStorage = localStorage

window.localStorage.setItem('user', JSON.stringify({userName: 'test'}))
describe('Message Pane Conversation Users Component', () => {

  it('renders as expected', () => {
    const tree = renderer
    .create(<ConversationUsers />)
    .toJSON();
    expect(tree).toMatchSnapshot();  
  })

  it('renders with Id', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ConversationUsers activeConversation={123}/> , div);
    ReactDOM.unmountComponentAtNode(div);
  })
})