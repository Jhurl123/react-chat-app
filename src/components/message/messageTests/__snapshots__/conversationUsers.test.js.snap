// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Pane Conversation Users Component renders as expected 1`] = `
<div
  className="makeStyles-users-1"
>
  <p>
    
  </p>
</div>
`;
